examples/stream-to-file-example.ts

```diff
@@ -0,0 +1,35 @@
+import 'dotenv/config'
+import { RingApi } from '../api'
+import fs = require('fs');
+
+/**
+ * An example using the RingCamera's recordLiveVideoToFile function.
+ */
+async function example() {
+  const ringApi = new RingApi({
+      // Replace with your ring email/password
+      email: process.env.RING_EMAIL!,
+      password: process.env.RING_PASS!,
+      // Refresh token is used when 2fa is on
+      refreshToken: process.env.RING_REFRESH_TOKEN!
+    }),
+    [camera] = await ringApi.getCameras()
+
+  if (!camera) {
+    console.log('No cameras found')
+    return
+  }
+  // Get the absolute path to the examples directory.
+  const dirname = __dirname.split("/").slice(0,-2).join("/")+"/examples/recordings/";
```

Comment:
I would use `path.join` here instead since windows uses `\` for separator

Reply:
People still use windows? Jk I'll fix it up.

Reply:
Addressed in subsequent commits.